### DISCUSSION TITLE
Question: how to convert an icon to System.Drawing.Icon?

### DISCUSSION THREAD

**Discussion by alonam** [MahApps/MahApps.Metro.IconPacks#289] [Q&A] [answered]
Hi
Is there a way to convert an icon to System.Drawing.Icon at runtime?
Thank you!

**Comment by punker76** [comment, marked as answer]
@AlonAm Maybe this code from [wpf-notifyicon](https://github.com/hardcodet/wpf-notifyicon) could be the solution

https://github.com/hardcodet/wpf-notifyicon/blob/4b89ede756b750e13c112dabb7afb3c42b3cb8cd/src/NotifyIconWpf/Util.cs#L159-L185

**Reply by alonam** [reply]
I see that it takes an ImageSource as input. I'll try to use [this](https://github.com/MahApps/MahApps.Metro.IconPacks/blob/8135ceb9a9be17ffa91d7d608ec1c9e52ad73ebd/src/MahApps.Metro.IconPacks.Core/Converter/PackIconKindToImageConverterBase.cs) to convert from the Enums programmatically (no XAML)
Thanks!
